app/controllers/api/v1/reset_passwords_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class ResetPasswordsController < ApplicationController
+
+      def index
```

Comment:
Is this needed...
You will be returning the headers you sent in the request?

Reply:
Yes, there are two request. The first request is a POST where you send the email and the redirect url. Then the API retorn the headers to change your password in the next PUT request. In this case, we don't use front end, so I considered necessary to put these headers in a controller in order to access to them when you click the button in the email